src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+/**
+ * Represents the UI
+ */
+public class Ui {
```

Comment:
this class should also take care of the parts where you call `System.out.println` manually, as this is supposed to be a representation of the terminal user interface